src/main/java/mate/academy/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.BookDao;
+import mate.academy.model.Book;
+import mate.academy.service.BookService;
+
+public class BookServiceImpl implements BookService {
+    private final BookDao bookDao;
+
+    public BookServiceImpl(BookDao bookDao) {
+        this.bookDao = bookDao;
+    }
+
+    @Override
+    public Book save(Book book) {
+        return bookDao.create(book);
+    }
+
+    @Override
+    public Book get(Long id) {
+        return bookDao.findById(id)
+                .orElseThrow(() -> new RuntimeException("Not found book with id: " + id));
```

Comment:
The use of RuntimeException is too generic here. According to best practices, you should use more informative and specific exceptions. Instead of RuntimeException, consider creating a custom exception that better describes the situation, such as 'BookNotFoundException'. This will improve the readability and maintainability of your code.